src/test/java/com/wikia/webdriver/testcases/mercurytests/HTMLTitleTests.java

```diff
@@ -122,13 +132,63 @@ private void init() {
   public void mercury_htmlTitleSet() {
     for (String[] testCase : testCases) {
       String testUrl = urlBuilder.appendQueryStringToURL
-              (new Page(testCase[0], testCase[1]).getUrl(),
-                      "cb=" + DateTime.now().getMillis());
+          (new Page(testCase[0], testCase[1]).getUrl(),
+           "cb=" + DateTime.now().getMillis());
 
       navigate.toUrl(testUrl);
       String actualTitle = head.getDocumentTitle();
 
       Assertion.assertEquals(actualTitle, testCase[2]);
     }
   }
+
+  @Test(groups = "seo_guard")
+  public void seo_guard() {
+    String originalEnv = Configuration.getEnv();
+
+    try {
+      PrintWriter out = new PrintWriter("./logs/seo-guard.txt");
+      PrintWriter out2 = new PrintWriter("./logs/seo-guard2.txt");
+
+      for (String[] testCase : testCases) {
+        navigate.toUrl(new Page(testCase[0], testCase[1]).getUrl());
+        new Wait(driver).forElementVisible(By.cssSelector(".side-nav-toggle-2016"));
+        this.pushMetaTagsToFile(out);
+
+        if (!originalEnv.equals("prod")) {
+          Configuration.setTestValue("env", "prod");
+
+          navigate.toUrl(new Page(testCase[0], testCase[1]).getUrl());
+          new Wait(driver).forElementVisible(By.cssSelector(".side-nav-toggle-2016"));
+          this.pushMetaTagsToFile(out2);
+
+          Configuration.setTestValue("env", originalEnv);
+        }
+      }
+
+      out.close();
+      out2.close();
+    } catch (FileNotFoundException e) {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00108)